syntax/std_seq_array_helper.go

```diff
@@ -108,22 +127,25 @@ func arrayHasPrefix(prefix rel.Value, subject rel.Array) rel.Value {
 				break
 			}
 		} else {
-			return rel.NewBool(false)
+			return rel.NewBool(false), nil
 		}
 	}
 
-	return rel.NewBool(true)
+	return rel.NewBool(true), nil
 }
 
 // Check if array subject ends with suffix.
-func arrayHasSuffix(suffix rel.Value, subject rel.Array) rel.Value {
-	suffixArray := convert2Array(suffix)
+func arrayHasSuffix(suffix rel.Value, subject rel.Array) (rel.Value, error) {
+	suffixArray, is := rel.AsArray(suffix)
+	if !is {
+		return nil, fmt.Errorf("has_suffix: suffix not an array: %v", suffix)
```

Comment:
error message isn't very consistent, some only show the function name others show the full name.

Reply:
Done.